pymtl3/passes/rtlir/rtype/RTLIRDataType.py

```diff
@@ -224,7 +222,7 @@ def _get_rtlir_dtype_struct( obj ):
     # Use user-provided pack order
     pack_order = []
     if hasattr( cls, "fields" ) and cls.fields != []:
-      assert len(cls.fields) == len(all_properties.keys()), \
+      assert len(cls.fields) == len(list(all_properties.keys())), \
```

Comment:
`len(list(all_properties.keys()))` == `len(all_properties)`, and the latter is easier to read (and faster again)